vast-perf.sh

```diff
@@ -380,100 +381,102 @@ EOF
 #       done
 
 
-while [ $# -gt 0 ]; do
-  case "$1" in
-    --vms=*)
-      mVIP="${1#*=}"
-      ;;
-    --proxy=*)
-      PROXY="${1#*=}"
-      ;;
-    --export=*)
-      NFSEXPORT="${1#*=}"
-      ;;
-    --test=*)
-      TEST="${1#*=}"
-      ;;
-    --runtime=*)
-      RUNTIME="${1#*=}"
-      RUNTIMEWRITE=$RUNTIME
-      ;;
-    --proto=*)
-      PROTO="${1#*=}"
-      ;;
-    --jobs=*)
-      JOBS="${1#*=}"
-      ;;
-    --size=*)
-      SIZE="${1#*=}"
-      ;;
-    --mix=*)
-      MIX="${1#*=}"
-      ;;
-    --block-size=*)
-      BLOCKSIZE="${1#*=}"
-      ;;
-    --pool=*)
-      POOL="${1#*=}"
-      ;;
-    --alt-pool=*)
-      ALT_POOL="${1#*=}"
-      ;;
-    --path=*)
-      REMOTE_PATH="${1#*=}"
-      ;;
-    --binary=*)
-      FIO_BIN="${1#*=}"
-      ;;
-    --mountpoint=*)
-      MOUNT="${1#*=}"
-      ;;
-    --delete=*)
-      DELETE_ALL="${1#*=}"
-      ;;
-    --ioengine=*)
-      ioengine="${1#*=}"
-      ;;
-    --iodepth=*)
-      iodepth="${1#*=}"
-      ;;
-    --direct=*)
-      DIRECT="${1#*=}"
-      ;;
-    --usevms=*)
-      USE_VMS="${1#*=}"
-      ;;
-    --distmode=*)
-      CN_DIST_MODE="${1#*=}"
-      ;;
-    --avoid-isl=*)
-      CN_AVOID_ISL="${1#*=}"
-      ;;
-    --loopback=*)
-      LOOPBACK="${1#*=}"
-      ;;
-    --user=*)
-      ADMINUSER="${1#*=}"
-      ;;
-    --password=*)
-      ADMINPASSWORD="${1#*=}"
-      ;;
-    --verbose=*)
-      VERBOSE="${1#*=}"
-      ;;
-    --vipfile=*)
-      VIPFILE="${1#*=}"
-      ;;
-    --help=*)
-      HELPME="true"
-      ;;
-    *)
-      showusage
-      exit 1
-  esac
-  shift
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --name=*) file_name="${1#*=}" ;;
```

Comment:
`--job-name` would be a better name for this argument, placed near `--jobs` below. It also needs to be added to the usage statement.